Source/ProgressionSystemRuntime/Public/Data/PSDataAsset.h

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "PSDataAsset.generated.h"
+
+/**
+ * Contains all progression assets used in the module 
+ */
+UCLASS()
+class PROGRESSIONSYSTEMRUNTIME_API UPSDataAsset : public UDataAsset
+{
+	GENERATED_BODY()
+
+public:
+	/** Returns the settings data asset. */
+	static const UPSDataAsset& Get() { return *GetDefault<ThisClass>(); }
+
+	/** Returns the Progression Data Table
+	 * @see UProgressionSystemDataAsset::ProgressionDataTableInternal */
+	UFUNCTION(BlueprintPure, Category = "C++")
+	FORCEINLINE UDataTable* GetProgressionDataTable() const { return ProgressionDataTableInternal; }
```

Comment:
Data Table should never be changed in runtime, so please return only `const` Data Table pointer:
`-->[const]<-- FORCEINLINE UDataTable* GetProgressionDataTable() const { return ProgressionDataTableInternal; }`

Reply:
added const. Hope lessons learned: data tables always const never changed during runtime